web/test/pages/dashboard.test.tsx

```diff
@@ -118,6 +120,60 @@ describe("dashboard page", () => {
     expect(screen.getByTestId("dashboard-alerts")).toBeInTheDocument();
   });
 
+  it("renders not-registered card operatingPhase is GUEST_MODE and businessPersona is STARTING", () => {
+    const business = generateBusiness({
+      profileData: generateProfileData({
+        businessPersona: "STARTING",
+        operatingPhase: "GUEST_MODE",
+      }),
+      preferences: generatePreferences({
+        visibleSidebarCards: ["not-registered", "not-registered-existing-account"],
```

Comment:
Shouldn't this be only not-registered?

Reply:
I could 100% be reading the story wrong, but my impression was that only not-registered card for Nexus and Poppy while only not-registered-existing-account for Oscar?

Reply:
I updated the test to coincide with starting users with an empty `visibleSidebarCards` array and the conditional logic to display the relevant card based on persona and operatingPhase.